test/new_relic/agent/attribute_filter_test.rb

```diff
@@ -94,16 +94,24 @@ def test_resque_capture_params_false_adds_exclude_rule_for_request_parameters
     end
 
     def test_resque_capture_params_true_allows_request_params_for_traces_and_errors
-      with_config(:'resque.capture_params' => true) do
+      with_config(:'attributes.include' => ['job.resque.args.*']) do
         filter = AttributeFilter.new(NewRelic::Agent.config)
         result = filter.apply('job.resque.args.*', AttributeFilter::DST_NONE)
 
-        assert_destinations %w[transaction_tracer error_collector], result
+        expected_destinations = %w[
```

Comment:
Expected destinations changed because the deprecated config was super old and didn't work with events